app/models/chess_piece.rb

```diff
@@ -2,6 +2,14 @@ class ChessPiece < ApplicationRecord
 
   belongs_to :game
 
+  def is_white?
```

Comment:
You could combine both of these method into one. If you call is_white? and it returns false then you know it must be black. Doesn't really matter though